src/widgets/DataTable/DataTable.js

```diff
@@ -106,17 +112,19 @@ const DataTable = React.memo(
 
     return (
       <RefContext.Provider value={contextValue}>
-        {renderHeader && renderHeader()}
-        <VirtualizedList
-          ref={virtualizedListRef}
-          keyboardDismissMode="none"
-          data={data}
-          keyboardShouldPersistTaps="always"
-          style={style}
-          ListFooterComponent={<KeyboardSpacing />}
-          renderItem={renderItem}
-          {...otherProps}
-        />
+        <View style={{ flex: 1 }}>
+          {renderHeader && renderHeader()}
+          <VirtualizedList
+            ref={virtualizedListRef}
+            keyboardDismissMode="none"
+            data={data}
+            keyboardShouldPersistTaps="always"
+            style={style}
+            ListFooterComponent={<KeyboardSpacing />}
+            renderItem={renderItem}
+            {...otherProps}
+          />
+        </View>
```

Comment:
this was for putting the table in `Paper` which wasn''t playing nice. Seems to be OK everywhere